docs/01_nodeos/03_plugins/producer_plugin/index.md

```diff
@@ -72,20 +72,9 @@ Config Options for eosio::producer_plugin:
                                         can extend during low usage (only
                                         enforced subjectively; use 1000 to not
                                         enforce any limit)
-  --produce-time-offset-us arg (=0)     Offset of non last block producing time
-                                        in microseconds. Valid range 0 ..
-                                        -block_time_interval.
-  --last-block-time-offset-us arg (=-200000)
-                                        Offset of last block producing time in
-                                        microseconds. Valid range 0 ..
-                                        -block_time_interval.
-  --cpu-effort-percent arg (=80)        Percentage of cpu block production time
-                                        used to produce block. Whole number
-                                        percentages, e.g. 80 for 80%
-  --last-block-cpu-effort-percent arg (=80)
-                                        Percentage of cpu block production time
-                                        used to produce last block. Whole
-                                        number percentages, e.g. 80 for 80%
+  --produce-block-offset-ms arg (=450)  The number of milliseconds early the
```

Comment:
As this is a new option, we have freedom to pick an intuitive name. The `produce` indicates it is a producer_plugin option, so it is better be `producer`; `produce` is a verb; it does not sound right when you read `produce block offset` in whole. This option is for a whole round; `block` is misleading in the name. `offset` is ambiguous; what does it refers to?

Reply:
Maybe default should be 480, as it is divisible by 12.

Reply:
The name was suggested by community. Do you have an alternative name you would suggest?
No reason for it to be evenly divided by 12 ,but maybe we should round up in the division by 12 in `set_produce_block_offset` so 450 becomes 456 instead of 444.

Reply:
Updated to use `std::ceil` on `produce-block-offset-ms / 12.0`

Reply:
OK, just keep the name suggested by the community. After reading the code, I got used to it.

Reply:
Just realized there was no reason to round and lose precision. Updated to calculate in microseconds.